poc/vdaf_mastic.py

```diff
@@ -96,6 +96,11 @@ def is_valid(cls, agg_param, previous_agg_params):
             # One solution is to make the first level a parameter of the VDAF.
             # This is probably a good idea anyway, since it's a trade-off the
             # Aggregators will probably want to agree on anyway.
+            #
+            # Another solution is to change the formatting of agg_param to
```

Comment:
To be honest I think yours is a much better solution than mine. Feel free to take over the `TODO`.